src/styles/components/header.scss

```diff
@@ -1,2 +1,78 @@
 .header {
+    background: var(--color-background-darker);
+    font-size: 2rem;
+    font-weight: 600;
+    border-bottom: 1px solid #5d5c61;
+    
+
+    &__content{
+        // height: 50px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    &__content-item{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-shrink: 5;
+        margin-left: 40px;
+        padding: 15px 0;
+    }
+
+    &__content-item:nth-of-type(2){
```

Comment:
this may be a risky selector as content changes since it seems to be selecting the end div lets use `:last-child`

Reply:
I agree. I changed it now so it will select that second grouping of the header content.